### PR TITLE
Add template readme for HW projects

### PR DESCRIPTION
**Pull request by tokenstar**
Adding a template readme for hardware projects, and a few minor fixes

### CODE REVIEW

**Review by cawhitworth — APPROVED** [review]

**Review by msrcstuart — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This repository contains a collection of inspirational, unmaintained [Azure Sphere](https://www.microsoft.com/azure-sphere/) software samples and hardware designs ready to be reused.
 
-Please also see the official Azure Sphere Sample and Azure Sphere Hardware Designs for maintained examples:
+Please also see the official Azure Sphere Samples and Azure Sphere Hardware Designs repos for maintained examples:
```
Comment (pennyo) [inline README.md:5]: Suggest inverting this sentence so reader knows what it's about first:

For maintained samples, see ...

File: Templates/hardware-readme-template.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<!--- This template should be used if the project contains hardware designs. If the project contains both hardware and software designs, please also refer to the software template and incorporate relevant sections. -->
+
```
Comment (pennyo) [inline Templates/hardware-readme-template.md:2]: Name of the software template?  

We're saying here that we want only one readme, so would it be clearer/easier for contributors to have only one template and tell people to delete the irrelevant parts?

**Review by pennyo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tokenstar) [reply README.md]: done

File: Templates/hardware-readme-template.md (hunk 2 above)
Comment (tokenstar) [reply Templates/hardware-readme-template.md:2]: as we expect many projects might be software-only or hardware-only, lets leave it as having two readmes for now.

**Pull request merged**